toolkit/scripts/check_static_glibc.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from pyrpm.spec import Spec
+from os.path import dirname, realpath
+
+import argparse
+import re
+import sys
+
+def get_glibc_version():
+    glibc_spec = Spec.from_file("SPECS/glibc/glibc.spec")
+    epoch_prefix = "{}:".format(glibc_spec.epoch) if glibc_spec.epoch else ""
+    release_without_dist = glibc_spec.release.replace("%{?dist}", "")
+    release_suffix = "-{}".format(release_without_dist)
+    glibc_version = epoch_prefix + glibc_spec.version + release_suffix
+    return glibc_version
+
+def check_spec(path, glibc_version):
+    spec = Spec.from_file(path)
+    for br in spec.build_requires:
+        if br.name == "glibc-static":
+            issues = []
+            if not br.version:
+                issues.append(f"  * Does not specify a minimum version for its `glibc-static` BuildRequires")
+            if br.version and br.version != glibc_version:
+                issues.append(f"  * Does not specify the latest `glibc-static` version in its BuildRequires")
+            if br.operator and br.operator != ">=":
+                issues.append(f"  * Uses bad BuildRequire `glibc-static` comparison operator `{br.operator}`")
+
+            if issues:
+                print(f"Specfile {spec.name} (at {path}):")
+                for issue in issues:
+                    print(issue)
+                print(f"  Suggestion: Use `glibc-static >= {glibc_version}%{{?dist}}`")
```

Comment:
At this point it's probably no longer a suggestion.;)

```suggestion
                print(f"  Use: `glibc-static >= {glibc_version}%{{?dist}}`")
```

Reply:
"Strongly suggest" 😝